### PR TITLE
fix: 1022 welcome description overlap

### LINKED ISSUE #1022: Welcome description overlapping on screen less than 5"
**Actual Behaviour / Screenshots**
screenshot_2017-10-09-16-13-08

**Currently happening**
Less than 5" Screen, Welcome screen description text overlapping TabLayout

**Expected Behaviour / Screenshots**
screenshot_2017-10-09-18-39-07

**Would you like to work on the issue?**
Yes, Let's discuss if anyone having any other idea for this design else please allow me to make PR for this issue.

### PR DESCRIPTION
**Pull request by anandwana001**
Fixes #1022 

**Screenshots**
screenshot_2017-10-09-22-03-52
screenshot_2017-10-09-16-13-08

### CODE REVIEW

**Comment by anandwana001** [comment]
@chiragw15 could you please review this!!

**Comment by amitiwary999** [comment]
@anandwana001 can you provide an updated screenshot.

**Comment by anandwana001** [comment]
@amitiwary999 first screenshot in the description is the updated one, which show the overlapping issue solved as in the second screenshot in description.

**Comment by anandwana001** [comment]
@chiragw15 Could you please review and merge this if it is proper!!

File: app/src/main/res/layout/welcome_slide4.xml [hunk 1]
```diff
@@ -22,20 +21,19 @@
...
 
     <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center"
-        android:gravity="center"
-        android:padding="@dimen/intro_desc_padding"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+		android:layout_weight=".2"
```
Comment (chiragw15) [inline app/src/main/res/layout/welcome_slide4.xml]: Can you indent this please

**Pull request merged**